client/service/src/builder.rs

```diff
@@ -813,6 +813,12 @@ pub struct BuildNetworkParams<'a, TBl: BlockT, TExPool, TImpQu, TCl> {
 	pub finality_proof_request_builder: Option<BoxFinalityProofRequestBuilder<TBl>>,
 	/// An optional, shared finality proof request provider.
 	pub finality_proof_provider: Option<Arc<dyn FinalityProofProvider<TBl>>>,
+	/// Shared GRANDPA and BABE items that are used to generate sync states to be used by light
```

Comment:
What is when someone isn't using BABE or GRANDPA? Can't this be an optional RPC that we don't hard code here?

Reply:
Done